cognite_toolkit/cdf_tk/load.py

```diff
@@ -374,9 +382,12 @@ class AuthLoader(Loader[int, Group, GroupList]):
     )
 
     def __init__(
-        self, client: CogniteClient, target_scopes: Literal["all", "resource_scoped_only", "all_scoped_only"] = "all"
+        self,
+        client: CogniteClient,
+        mode: Literal["load", "all_only"] = "load",
+        target_scopes: Literal["all", "resource_scoped_only", "all_scoped_only"] = "all",
```

Comment:
`target_scopes` is obsolete if you start to use `mode`. However, I think you should keep `target_scope` and remove the `mode`.